pkg/cmd/template/schema_test.go

```diff
@@ -602,7 +662,29 @@ vpc:
 		filesToProcess := files.NewSortedFiles([]*files.File{
 			files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
 		})
-		expectedErr := "Array items cannot be annotated with #@schema/nullable (schema.yml:6). If this behaviour would be valuable, please submit an issue on https://github.com/vmware-tanzu/carvel-ytt"
+		expectedErr := `
+schema.yml:6 | - 0
+             |   ^^^
+             |  found: array item with an unexpected annotation
+             |  (hint: array items cannot be annotated with #@schema/nullable, if this behaviour would be valuable, please submit an issue on https://github.com/vmware-tanzu/carvel-ytt)`
+
+		assertYTTWorkflowFailsWithErrorMessage(t, filesToProcess, expectedErr)
+	})
+	t.Run("null value gives hint about nullable annotation", func(t *testing.T) {
```

Comment:
This test seemed identical to the one below "null given as a value", maybe this test was used to TDD the hint? Do we need both of these tests?